.moban.dt/core-setup.py.jj2

```diff
@@ -0,0 +1,49 @@
+{% extends 'coala-setup.py.jj2'%}
+
+{%block local_import_block%}
+import datetime
+from os import getenv
+
+import setuptools.command.build_py
+
+{%endblock%}
+
+{%block local_exec_block_1%}
+assert_supported_version()
+
+
+class BuildPyCommand(setuptools.command.build_py.build_py):
+
+    def run(self):
+        if platform.system() != 'Windows':
```

Comment:
what do we do if it is windows?

Reply:
Nothing.  Note that this template re-creates [the *existing* setup.py](https://github.com/coala/coala/blob/308e1e3/setup.py), and those lines were last modified in https://github.com/coala/coala/commit/e6bd2d3f6395ccc094e31a01402962fb60d40c8e.